client/ayon_core/pipeline/context_tools.py

```diff
@@ -384,42 +384,67 @@ def get_current_project_entity(fields=None):
     return ayon_api.get_project(project_name, fields=fields)
 
 
-def get_current_project_folder(folder_path=None, folder_id=None, fields=None):
+def get_current_folder_entity(fields=None):
     """Helper function to get folder entity based on current context.
 
     This function should be called only in process where host is installed.
 
-    Folder is found out based on passed folder path or id (not both). Folder
-    path is not used for filtering if folder id is passed. When both
-    folder path and id are missing then current folder path is used.
+    Folder is found out based on current project name and folder path.
```

Comment:
I think I prefer a wording like this:
```suggestion
    Folder is based on current context project name and folder path.
```